schemas/app-sre/saas-file-target-1.yml

```diff
@@ -2,72 +2,129 @@
+            type: array
+            items:
+              type: string
+            description: |
+              A list of JSON path selectors to exclude namespaces.
+        required:
+        - include
+
   ref:
     type: string
     pattern: '^([0-9a-f]{40}|master|main|internal|stable|release-4\.\d+)$'
+    description: |
+      The git reference (commit SHA or branch name) to deploy. For production
+      deployments, always use a commit SHA to ensure reproducibility.
+
   promotion:
     type: object
+    description: |
+      Promotion configuration for this target. Defines auto-promotion,
+      gating, publishing, and scheduling for progressive delivery. Allows
+      advanced deployment strategies such as soak days and scheduled windows.
     additionalProperties: false
     properties:
       auto:
         type: boolean
-      redeployOnPublisherConfigChange:
         description: |
-          Whether to redeploy if a config change happened in a publisher. This will set
-          target_config_hash in promotion_data. This is mainly intended for test jobs.
+          If true, enables automated promotion for this target. The system
+          will automatically update the ref when promotion criteria are met.
+      redeployOnPublisherConfigChange:
         type: boolean
+        description: |
+          If true, redeploys this target if a configuration change occurs
+          in a publisher. Useful for test jobs and validation workflows.
       publish:
         type: array
         items:
           type: string
         minItems: 1
+        description: |
+          A list of channels to publish successful deployments to. Other
+          targets can subscribe to these channels for gated promotions.
       subscribe:
         type: array
         items:
           type: string
         minItems: 1
+        description: |
+          A list of channels to subscribe to for gating this deployment.
+          Deployment will only proceed if all subscribed channels have
+          published a successful deployment for the same ref.
       soakDays:
         type: integer
-        description: number of days to wait for promotion
+        description: |
+          The number of accumulated days a ref must have been running across
+          all publishers before promotion. Used for progressive rollouts.
       schedule:
         type: string
-        description: Same as AUS schedule - crontab expression to optionally restrict deployment windows
         pattern: '(((\d+,)+\d+|(\d+(\/|-)\d+)|\d+|\*) ?){5}'
+        description: |
+          Cron expression to restrict deployment windows for auto promotions.
+          Defines when auto-promotion merge requests can be opened.
       promotion_data:
         type: array
+        description: |
+          Data managed by integrations, relative to what triggered a promotion.
+          Used to track configuration hashes and other metadata for advanced
+          promotion workflows.
```

Comment:
usually this should not be edited manually

Reply:
`Data managed by integrations` - I think this phrase covers the concept? Is it unclear?